charts/refarch-templates/values-example.yaml

```
@@ -79,7 +79,7 @@ refarch-templates:
       volumeMounts:
         *cacertsVolumeMount
 
-# Sample Gateway configuration, requires additional chart dependency on https://github.com/it-at-m/helm-charts/tree/main/charts/refarch-gateway
+# Sample Gateway configuration, which is enabled by default. For further configuration option view the chart dependency on https://github.com/it-at-m/helm-charts/tree/main/charts/refarch-gateway
 refarch-gateway:
   envFrom:
```

```suggestion
  enable: true
  envFrom:
```

ich hätte es halt so rein

---

Mhh man brauch es nicht - ist ja enabled on-default. Habe dies auch in der Doku beschrieben. Ich würde dies rauslassen, könnte sonst verwirren.

---

dann ist halt das drin für den enduser. und man muss nicht in der doku nachlesen